src/test/regress/expected/multi_partitioning_1.out

```diff
@@ -0,0 +1,1939 @@
+--
```

Comment:
I think we should drop adding the file right now, and do it whenever we actually need it on pg12 branch?

Reply:
This is an expected result for pg11

Reply:
Wait, then why do we need to add now?

Reply:
Fixed. This isn't an addition, it's renaming `multi_partitioning_9.out` to `multi_partitioning_0.out`. This file's a bit of a mess since testing framework changes leaves us having 4 versions once we add in `_2.out`